XamlControlsGallery/ControlPages/CommandBarPage.xaml.cs

```diff
@@ -18,8 +19,31 @@ namespace AppUIBasics.ControlPages
     /// <summary>
     /// An empty page that can be used on its own or navigated to within a Frame.
     /// </summary>
-    public sealed partial class CommandBarPage : Page
+    public sealed partial class CommandBarPage : Page, INotifyPropertyChanged
```

Comment:
I may be completely wrong, but I didn't think you needed to implement INotifyPropertyChanged to use x:Bind (as opposed to Binding). What happens when you just declare a public MultipleButtons bool, without the getter and setter?